plugins/woocommerce-blocks/assets/js/base/utils/address.ts

```diff
@@ -194,7 +205,17 @@ export const isAddressComplete = (
 		address.country
 	);
 
-	return addressForm.every(
+	// Filter the address form so only fields from the keysToCheck arg remain, if that arg is empty, then default to the
+	// full address form.
+	const filteredAddressForm =
+		keysToCheck.length > 0
+			? ( Object.values( addressForm ) as KeyedFormField[] ).filter(
+					( { key } ) =>
+						keysToCheck.includes( key as keyof CoreAddress )
+			  )
+			: addressForm;
+
+	return filteredAddressForm.every(
```

Comment:
Required because the address was never considered "complete" if accessing from a fresh session and never entering fields beyond city, state, postcode, and country

Reply:
Does this take localization into account?

Reply:
In what sense? The form is statically defined here: https://github.com/woocommerce/woocommerce/blob/9ad83238517a43e39a28f6a0c9a063e3dfc42ac6/plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/shipping-calculator/index.tsx#L30

This is to check whether the shipping calculator form is complete, ignoring other required fields the user didn't get to enter yet.

Reply:
This list still goes via locale, I tested it and it's working fine.